tests/db_tests/postgres.test.js

```diff
@@ -0,0 +1,28 @@
+const db = require('../../database/postgres.js');
+
+let isErr = false;
+let queryData;
+
+const queryCallback = (err, data) => {
+  if (err) {
+    isErr = true;
+  } else if (data) {
+    queryData = data;
+  }
+}
+
+describe('getRestaurant', () => {
+  
+  it('invokes the callback with data for a successful query', () => {
+    const goodQuery = db.getRestaurant(1, queryCallback);
+    expect(queryData).toBeDefined
+  })
+
+  queryData = undefined;
+  
+  it('invokes the callback with data for a successful query', () => {
+    const goodQuery = db.getRestaurant(1, queryCallback);
+    expect(isErr).toBe(true);
```

Comment:
Test name should be changed because it looks like you're trying to test a bad query.
How are you getting an error message to set isErr to true? Isn't id: 1 in your db?